reactjs-dallas/reactjs-dallas-site
# [Events] Adds a better way to update/show events

# Issues

Closes #2 

# The issue

As a user, I should see past and future events.

# The solution

Use a shared component to map over the data and show a consistent UI for all past/future events.

Also:
- Adds material-ui...pretty interesting way to get it to work. 🤓 

# Before

![image](https://user-images.githubusercontent.com/11624407/54315091-4de3a700-45ab-11e9-81d2-27e234356e5f.png)

# After 

![reactjs-dallas--events-new](https://user-images.githubusercontent.com/11624407/54315206-7ec3dc00-45ab-11e9-8bba-20a2df564e6e.gif)

# To test

- Get this code.
- Run it locally → `yarn && yarn develop`
- Make sure that it looks stellar. ✨ 

-----------
<img src="http://drumsensei.com/warehouse/images/oldtimeyparrot.gif"/>
